lib/cloudregister/registerutils.py

```diff
@@ -274,17 +287,20 @@ def fetch_smt_data(cfg, proxies):
                 'Getting update server information, attempt %d' % retry_cnt
             )
             for srv in region_servers:
-                srvName = srv.strip()
-                logging.info('\tUsing region server: %s' % srvName)
+                logging.info('\tUsing region server: %s' % srv)
```

Comment:
There needs to be a test of the network configuration, if the network is not configured for IPv6 then the IPv6 addresses in the list should be skipped.

Reply:
Added in c9754a9 using `socket.has_ipv6` [1]

[1] https://docs.python.org/3/library/socket.html#socket.has_ipv6